src/components/Mana/Mana.css

```diff
@@ -18,7 +19,7 @@
   background-size: cover;
 }
 
-.dcl.mana + .dcl.mana {
+.dcl.mana+.dcl.mana {
```

Comment:
Having them together does not affect the style right ?

For example, its different having `.something .something` than having `.something.something`